lib/api/protocol/moveTo.js

```diff
@@ -22,16 +24,16 @@ const ProtocolAction = require('./_base-action.js');
  */
 module.exports = class Command extends ProtocolAction {
 
-  command(webElementId, xoffset, yoffset, callback = function() {}) {
+  command(webElementId, xoffset=0, yoffset=0, callback = function() {}) {
    
-    if (arguments.length >= 3 && typeof arguments[0] === 'string' || arguments[0] === null) {
-      if (webElementId === null) {
+    if (typeof arguments[0] === 'string' || arguments[0] == null) {
+      if (webElementId == null) {
```

Comment:
We should also shift callback accordingly. Like for the command call `moveTo(<elementId>, ()=>console.log('callback function'))` the method will fail.